src/installer/pkg/sfx/Microsoft.NETCore.App/Directory.Build.props

```diff
@@ -3,6 +3,7 @@
 
   <PropertyGroup>
     <SharedFrameworkName>Microsoft.NETCore.App</SharedFrameworkName>
+    <SharedFrameworkName Condition="'$(PgoOptimize)' != ''">$(SharedFrameworkName).PGO</SharedFrameworkName>
```

Comment:
You should also mark the PGO package variants as non-Shipping.

You'll likely also need to include the `pgort*.dll` file in the platform manifest for the PGO-instrumented build.